cmd/notation/cert/list.go

```diff
@@ -22,6 +22,20 @@ func certListCommand(opts *certListOpts) *cobra.Command {
 		Use:     "list",
 		Aliases: []string{"ls"},
 		Short:   "List certificates in the trust store.",
+		Long: `List certificates in trust store
+
+Example - List all certificate files stored in the trust store
+  notation certificate ls
+
+Example - List all certificate files of trust store "acme-rockets"
+  notation cert ls --store "acme-rockets"
```

Comment:
replace "cert" to "certificate"

Reply:
Let's keep using the short alias in each command line example. It will be easy to type.